setup.md

```diff
@@ -3,4 +3,11 @@ layout: page
 title: Setup
 permalink: /setup/
 ---
-See the [workshop template page]({{ site.workshop_site }}) for setup instructions.
+# Software
+For this course you will need the UNIX shell, plus [SQLite3](http://www.sqlite.org/) or
+[DB Browser for SQLite](http://sqlitebrowser.org/).
+
+If you are running Linux, you will already have SQLite3 installed.
```

Comment:
This is not usually the case (in particular it's not on Ubuntu)

Reply:
Or may be change it to.

If you are running Linux, you may already have SQLite3 installed, if not run 
```
sudo apt-get update
sudo apt-get install sqlite3 libsqlite3-dev
```
and run `sqlite3 --version` to confirm the installation

Reply:
`apt install sqlite3` should be enough. I don't know if we should add instructions for other distributions...

Reply:
which distributions?